concurrency/README.md

```diff
@@ -36,4 +36,9 @@ The first PipelineRun should be canceled, and the second one should execute norm
 Supported strategies are "Cancel", "GracefullyCancel", and "GracefullyStop"
 (corresponding to canceling, gracefully canceling, and gracefully stopping a PipelineRun, respectively).
 The default strategy is "GracefullyCancel".
-If multiple ConcurrencyControls with different strategies apply to the same PipelineRun, concurrency controls will fail.
\ No newline at end of file
+If multiple ConcurrencyControls with different strategies apply to the same PipelineRun, concurrency controls will fail.
+
+### Configuration
+
+To restrict the concurrency webhook and controller to only modify PipelineRuns in a subset of namespaces,
+edit the "allowed-namespaces" field of the [concurrency configMap](./config/concurrency-config.yaml).
```

Comment:
do you mind noting that the list of namespaces is a comma-separated string with all namespaces

would this be a list of strings instead of one string with commas?

Reply:
Updated the readme. For a list instead of a comma separated string-- based on [this example](https://kubernetes.io/docs/concepts/configuration/configmap/#configmaps-and-pods), it seems like a comma separated string is the way to specify a list in a configmap, and it's also not clear that one way is easier than the other